models/wholeBrainSeg_Large_UNEST_segmentation/configs/metadata.json

```diff
@@ -0,0 +1,211 @@
+{
+    "schema": "https://github.com/Project-MONAI/MONAI-extra-test-data/releases/download/0.8.1/meta_schema_20220324.json",
+    "version": "0.1.1",
+    "changelog": {
+        "0.1.0": "complete the model package",
+        "0.1.1": "fixed the model state dict name"
```

Comment:
Hi @tangy5 , it seems you may modify the wrong place, this bundle is a new one, thus it may not need to add two change log, maybe 0.1.1 can be removed.
In addition, usually we add the new changelog in the top of the dict.

Reply:
Fixed as below: 
```
"version": "0.1.1",
    "changelog": {
        "0.1.1": "fixed the model state dict name",
        "0.1.0": "complete the model package"
    },
```
Thanks